db/migrate/20160719100630_create_wishlists.rb

```diff
@@ -0,0 +1,9 @@
+class CreateWishlists < ActiveRecord::Migration[5.0]
+  def change
+    create_table :wishlists do |t|
+      t.string :name
```

Comment:
For :name it's simply null: false, but what about :description, should I just omit it in migration? As we don't want to imply whether it's true or false?